avidemux_plugins/ADM_videoEncoder/x265/qt4/Q_x265.cpp

```diff
@@ -79,19 +79,13 @@ static const aspectRatio predefinedARs[]={
 
 #define NB_SAR sizeof(predefinedARs)/sizeof(aspectRatio)
 
-typedef struct
-{
-    int bdValue;
-    const char *bdString;
-}bdToken;
-
-static const bdToken listOfBitDepths[]={
+static const idcToken listOfBitDepths[]={
     {8,"8"},
     {10,"10"},
     {12,"12"}
 };
 
-#define NB_BITS sizeof(listOfBitDepths)/sizeof(bdToken)
+#define NB_BITS sizeof(listOfBitDepths)/sizeof(*listOfBitDepths)
```

Comment:
Using *listOfBitDepths will produce the right value if listOfBitDepths changes type.  There's a hard to find bug where someone forgets to change NB_XXX after changing the array type, but the code still compiles without any error or warning, and maybe works fine, until later when someone changes one of the types and the size becomes wrong.